src/Microsoft.Azure.WebJobs.Extensions.EventHubs/EventHubExtensionConfigProvider.cs

```diff
@@ -48,14 +48,11 @@ public void Initialize(ExtensionConfigContext context)
                 throw new ArgumentNullException(nameof(context));
             }
 
-            // TODO: Can we bind these during service setup?
-            // apply at eventProcessorOptions level (maxBatchSize, prefetchCount)
-
             EventProcessorOptions options = _options.GetOptions();
             options.SetExceptionHandler(ExceptionReceivedHandler);
-            _configuration.Bind("eventHub", options);
 
             // apply at config level (batchCheckpointFrequency)
+            // TODO: Revisit this... All configurable options should move to the Options type.
             _configuration.Bind("eventHub", _options);
```

Comment:
We need to cleanup the `EventHubConfiguration` type and remove the direct dependency the config provider has on `IConfiguration`. This class shouldn't be binding here.